### ISSUE TITLE
terminate called after throwing an instance of 'H5::FileIException'

### ISSUE THREAD

**Issue by pedrojlsilva** [anhttran/extreme_3d_faces#49]
`root@ea21a7cb96ec:/app/bin# ./TestBump -batch /shared/imList.txt /shared/3D/ /shared/shape /shared/bump /shared/bump ../3DMM_model/BaselFaceModel_mod.h5 ../dlib_model/shape_predictor_68_face_landmarks.dat /shared/imgs /shared/imgs/ 1
load ../3DMM_model/BaselFaceModel_mod.h5
terminate called after throwing an instance of 'H5::FileIException'
`

Hello, i'm facing this problem when a tried to run the TestBump code. Could you help me to find a solution? Thank you so much

**Comment by pedrojlsilva** [comment]
well, i'm using the docker image provided by you @anhttran and when I execute the 'python testBatchModel.py testImages.txt /shared' command, the last line of this scripts returns this failed status and calls a segmentation fault

**Comment by anhttran** [comment]
Dear,

Do you have the file `BaselFaceModel_mod.h5` in the `3DMM_model` folder? If not, please generate it following the Data Requirements section (steps 3-5).

Best regards,

**Comment by pedrojlsilva** [comment]
Thank you @anhttran ! 

I tried another solution, and I'll explain here. 

I'm using your docker version and reading the Readme file, was not clear to me that I had to regerate the `BaselFaceModel_mod.h5`. (If I missed anything, sorry about that!)

So, i installed the `h5py` using `pip install h5py` and executed the `trimBaselFace.py` inside the docker. After that, the `BaselFaceModel_mod.h5` was update with 2 new tags called `pairs` and `faces_fill` that did not exist before.

Well, Now I tested and it worked. Thanks for you quick answer!
